### PR TITLE
add gpu option

### PR DESCRIPTION
**Pull request by vikimark**
### What does this changes

add gpu option on translate module

### CODE REVIEW

**Comment by cstorm125** [comment]
LGTM. @wannaphong this is one of my Builders. Should we tell him to fix the PEP8 before merge?

**Comment by wannaphong** [comment]
> LGTM. @wannaphong this is one of my Builders. Should we tell him to fix the PEP8 before merge?

Yes, I think like you.

**Comment by wannaphong** [comment]
I don't sure that It can working with GPU by use CUDA_VISIBLE_DEVICES to set available GPU. If it can use by use CUDA_VISIBLE_DEVICES to set available GPU, I think maybe we needs pull request to update docs not add gpu parameter.

File: pythainlp/translate/core.py [hunk 1]
```diff
@@ -25,10 +25,11 @@ class Translate:
         th2en.translate("ฉันรักแมว")
         # output: I love cat.
     """
-    def __init__(self, src_lang: str, target_lang: str) -> None:
+    def __init__(self, src_lang: str, target_lang: str, use_gpu=False) -> None:
         """
         :param str src_lang: source language
-        :param str target_lang: target language
+        :param str target_lang: target
```
Comment (wannaphong) [inline pythainlp/translate/core.py]: Can you add use_gpu parameter to the docs?

File: pythainlp/translate/en_th.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ class EnThTranslator:
 
     Website: https://airesearch.in.th/releases/machine-translation-models/
     """
-    def __init__(self):
+    def __init__(self, use_gpu: bool):
```
Comment (wannaphong) [inline pythainlp/translate/en_th.py]: Can you add use_gpu parameter to the docs?

Comment (wannaphong) [inline pythainlp/translate/en_th.py]: I think it should has default is False.

**Review by wannaphong — CHANGES_REQUESTED** [review]
This is my comment.

**Comment by vikimark** [comment]
I've resolved all your comments. And should I add docs to the  __init__ of each class?

File: pythainlp/translate/core.py [hunk 3]
```diff
@@ -7,6 +7,7 @@ class Translate:
 
     :param str src_lang: source language
     :param str target_lang: target language
+    :param bool use_gpu: load model to gpu
```
Comment (wannaphong) [inline pythainlp/translate/core.py]: I think you should add default. example: `(Default is False)`

**Review by wannaphong — CHANGES_REQUESTED** [review]

File: pythainlp/translate/core.py (hunk 3 above)
Comment (vikimark) [reply pythainlp/translate/core.py]: Okay, added to all of them

**Review by wannaphong — APPROVED** [review]

**Pull request merged**